### ISSUE TITLE
QdtViz Export data option is not working

### ISSUE THREAD

**Issue by bandana2** [qlik-demo-team/qdt-components#251]
Hi Team,

When using the exportData in the QdtViz is not working and is showing 'https://undefined/' in the browser.
I am using the "qdt-components": "^3.0.2" from Npm.

below is my code:

 if (id) {
      QdtViz({
        element: filterRef.current,
        app: cApp,
        options: { id,height:350 },
        properties:{exportData: true}
      });

**Comment by evan-jasiewicz** [comment]
Im having the same issue

**Comment by Dhruv-Qlik** [comment]
Yeah, I have also faced the same issue? Have you resolved @bandana2 ? or still the same?
